Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM node:4
+
+ENV LOUNGE_VERSION 1.5.0
+ENV NODE_ENV production
+
+ARG user=lounge
+ARG group=lounge
+ARG uid=1000
+ARG gid=1000
+
+ENV LOUNGE_HOME "/home/lounge"
+ENV LOUNGE_DATA "${LOUNGE_HOME}/data"
+ENV LOUNGE_SRC "${LOUNGE_HOME}/src"
+
+# Create a non-root user for lounge to run in.
+RUN groupadd --gid ${gid} ${group} \
+      && useradd --home "${LOUNGE_HOME}" --create-home --uid ${uid} --gid ${gid} ${user}
+
+VOLUME "${LOUNGE_DATA}"
+RUN chown -R ${user} "${LOUNGE_DATA}"
+
+# Drop root.
+USER ${user}
+
+# Install thelounge.
+RUN mkdir -p "${LOUNGE_SRC}"
+WORKDIR "${LOUNGE_SRC}"
+RUN curl -L $(npm view thelounge@${LOUNGE_VERSION} dist.tarball) | tar --strip-components 1 -xzf -
```

Comment:
@williamboman, I get that you don't want the source installed at `./node_modules/thelounge` but [ we do recommend a global install](https://github.com/thelounge/lounge/blob/e6990e0fc7641d18a5bcbabddca1aacf2254ae52/package.json#L5) so you could use `npm install --global thelounge` and let the package manager (npm) decide where packages are installed, no?

I don't know if this solution is the most common way of doing it *in the Docker world*, but it seems to me that it's the cleanest way to do it considering The Lounge is software and not library. After all, I'm curious, why do you need to control where to install it?